.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+#trello
+TOKEN=secret-token
+APIKEY=api-key
```

Comment:
You have changed this to `API_KEY` in the code, so we need that change reflected here too.